Example/Pods/TextAnnotation/TextAnnotation/Classes/CustomSwitch.swift

```diff
@@ -0,0 +1,278 @@
+import Foundation
+
+public class CustomSwitch: NSControl {
+  public static let bundle = Bundle(for: CustomSwitch.self)
+  
+  public var tintColor = NSColor(calibratedRed: 0.243, green: 0.235, blue: 0.243, alpha: 1.0) {
+    didSet {
+      reloadLayer()
+    }
+  }
+  public var knobBackgroundColor = NSColor(calibratedWhite: 1.0, alpha: 1.0) {
+    didSet {
+      reloadLayer()
+    }
+  }
+  public var disabledBorderColor = NSColor(calibratedWhite: 0.0, alpha: 0.2) {
+    didSet {
+      reloadLayer()
+    }
+  }
+  public var disabledBackgroundColor = NSColor(calibratedRed: 0.243, green: 0.235, blue: 0.243, alpha: 1.0)  {
+    didSet {
+      reloadLayer()
+    }
+  }
+  public var inactiveBackgroundColor = NSColor(calibratedWhite: 0.0, alpha: 0.3) {
+    didSet {
+      reloadLayer()
+    }
+  }
+  public var animationDuration: TimeInterval = 0.4
+  public var inactiveIcon = bundle.image(forResource: "T_shadow_selected") {
+    didSet {
+      reloadLayer()
+    }
+  }
+  public var activeIcon = bundle.image(forResource: "T_shadow_unselected") {
+    didSet {
+      reloadLayer()
+    }
+  }
+  public var secondIcon = bundle.image(forResource:"T_outline") {
+    didSet {
+      reloadLayer()
+    }
+  }
+  
+  override public var isEnabled: Bool {
+    didSet {
+      reloadLayerAnimated(animated: true)
+    }
+  }
+  
+  lazy var kBorderLineWidth: CGFloat = 0
+  let kEnabledOpacity: Float = 1.0
+  let kDisabledOpacity: Float = 0.5
+  var dragEvents = 0
+  
+  public var isOn: Bool = false
+  public var isActive: Bool = false
+  public var hasDragged: Bool  = false
+  public var isDraggingTowardsOn: Bool = false
+  public var lockInteraction: Bool = false
+  public let rootLayer = CALayer()
+  public let backgroundLayer = CALayer()
+  public let knobLayer = CALayer()
+  public let icon1Layer = CALayer()
+  public let icon2Layer = CALayer()
+  
+  override public var acceptsFirstResponder: Bool {
+    get { true }
+  }
+  
+  required public init?(coder:NSCoder) {
+    super.init(coder: coder)
+    setup()
+  }
+  
+  convenience public init(size: CGSize) {
+    self.init(frame: NSRect(origin: .zero, size: size))
+  }
+  
+  override public init(frame: NSRect) {
+    super.init(frame: frame)
+    setup()
+  }
+  
+  internal func setup() {
+    isEnabled = true
+    setupLayers()
+    reloadLayerSize()
+    setupIcon()
+    reloadLayer()
+  }
+  
+  public func setOn(isOn: Bool, animated: Bool) {
+    if self.isOn != isOn {
+      self.isOn = isOn
+      reloadLayerAnimated(animated: animated)
+    }
+  }
+  
+  internal func setupIcon() {
+    guard let icon = inactiveIcon else {
+      return
+    }
+    
+    let bounds = knobLayer.bounds
+    let size = icon.size
+    icon1Layer.frame = NSMakeRect((bounds.width-size.width)/2, (bounds.height-size.height)/2, size.width, size.height)
```

Comment:
(bounds.width-size.width)/2 should be (bounds.width-size.width) / 2 here and in other similar cases.
@dworak Please don't forget to add space between the operator and operands.